ingestion/functions/parsing/peru/peru.py

```diff
@@ -16,6 +16,21 @@
     import parsing_lib
 
 
+import json
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), "geocoding_dictionaries.json")) as json_file:
```

Comment:
I would probably add some info here about what the dictionaries are and where we've gotten the information from.

Reply:
Will do once we've gotten collated geocoding info in a file in ingestion/functions as discussed yday

Reply:
You don't think it's also worth putting some info in the parser itself? see Argentina parser